src/components/cluster/detail/cluster_detail_node_pools_table.js

```diff
@@ -239,7 +216,7 @@ class ClusterDetailNodePoolsTable extends React.Component {
           </GridRowNodePoolsNodes>
           <GridRowNodePoolsHeaders>
             <span>ID</span>
-            <span>NAME</span>
+            <span style={{ paddingLeft: '8px' }}>NAME</span>
```

Comment:
In general, I'd like to avoid "inline" style, but I think in this case, it's a fine compromise. I think sometimes the benefit of having things 'closer together' can outweigh the rule.

Reply:
yeah, in this case this is a very specific thing for only this column. In such situations  I think that this is the best thing to do